server/src/capture.rs

```diff
@@ -111,6 +176,31 @@ pub struct EventCapture {
     db_client: Arc<Mutex<Client>>,
```

Comment:
Perhaps make this a `Arc<Mutex<Option<Client>>>`? If the config file is found and the db connections, the logging happens. Otherwise, no logging happens but  the code still runs. (Currently, it errors if without a connection/config file/etc.)